### PR TITLE
DOCSUP-11545: document the optimize_trivial_count_query setting

### PR DESCRIPTION
**Pull request by kirillikoff**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):

- Documentation for #7510

### CODE REVIEW

File: docs/en/operations/settings/settings.md [hunk 1]
```diff
@@ -1933,6 +1933,21 @@ Possible values:
 
 Default value: `0`.
 
+## optimize_trivial_count_query {#optimize-trivial-count-query}
+
+Enables or disables the optimization to trivial query `SELECT count () FROM table` using metadata from MergeTree. If you need to use row-level security, disable trivial count query optimization.
```
Comment (adevyatova) [inline docs/en/operations/settings/settings.md:1938]:
```suggestion
Enables or disables the optimization to trivial query `SELECT count () FROM table` using metadata from MergeTree. If you need to use row-level security, disable this setting.
```

File: docs/ru/sql-reference/aggregate-functions/reference/count.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ ClickHouse поддерживает следующие виды синтакси
 
 ClickHouse поддерживает синтаксис `COUNT(DISTINCT ...)`. Поведение этой конструкции зависит от настройки [count_distinct_implementation](../../../operations/settings/settings.md#settings-count_distinct_implementation). Она определяет, какая из функций [uniq\*](../../../sql-reference/aggregate-functions/reference/uniq.md#agg_function-uniq) используется для выполнения операции. По умолчанию — функция [uniqExact](../../../sql-reference/aggregate-functions/reference/uniqexact.md#agg_function-uniqexact).
 
-Запрос `SELECT count() FROM table` не оптимизирован, поскольку количество записей в таблице не хранится отдельно. Он выбирает небольшой столбец из таблицы и подсчитывает количество значений в нём.
+Запрос `SELECT count() FROM table` оптимизирован по умолчанию с использованием метаданных из MergeTree. Если в таблице установлена безопасность на уровне строк, отключите [optimize_trivial_count_query](../../../operations/settings/settings.md#optimize-trivial-count-query), установив `optimize_trivial_count_query = 0`.
```
Comment (adevyatova) [inline docs/ru/sql-reference/aggregate-functions/reference/count.md:32]:
```suggestion
Запрос `SELECT count() FROM table` оптимизирован по умолчанию с использованием метаданных из MergeTree. Если в таблице установлена безопасность на уровне строк, отключите оптимизацию при помощи настройки [optimize_trivial_count_query](../../../operations/settings/settings.md#optimize-trivial-count-query). 
```

File: docs/en/sql-reference/aggregate-functions/reference/count.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ In both cases the type of the returned value is [UInt64](../../../sql-reference/
 
 ClickHouse supports the `COUNT(DISTINCT ...)` syntax. The behavior of this construction depends on the [count_distinct_implementation](../../../operations/settings/settings.md#settings-count_distinct_implementation) setting. It defines which of the [uniq\*](../../../sql-reference/aggregate-functions/reference/uniq.md#agg_function-uniq) functions is used to perform the operation. The default is the [uniqExact](../../../sql-reference/aggregate-functions/reference/uniqexact.md#agg_function-uniqexact) function.
 
-The `SELECT count() FROM table` query is not optimized, because the number of entries in the table is not stored separately. It chooses a small column from the table and counts the number of values in it.
+The `SELECT count() FROM table` query is optimized by default using metadata from MergeTree. If you need to use row-level security, disable [optimize_trivial_count_query](../../../operations/settings/settings.md#optimize-trivial-count-query) setting by set `optimize_trivial_count_query = 0`.
```
Comment (adevyatova) [inline docs/en/sql-reference/aggregate-functions/reference/count.md:32]:
```suggestion
The `SELECT count() FROM table` query is optimized by default using metadata from MergeTree. If you need to use row-level security, disable optimization using the [optimize_trivial_count_query](../../../operations/settings/settings.md#optimize-trivial-count-query) setting.
```

**Review by kitaisreal — APPROVED** [review]

**Comment by Lloyd-Pottiger** [comment]
Until now, the Chinese docs have not been updated yet. 😭

**Pull request merged**